sagemaker-fundamentals/create-training-job/create_training_job_vpc.ipynb

```diff
@@ -0,0 +1,869 @@
+{
```

Comment:
<p>Typo, <code>subent</code> instead of <code>subnet</code>.</p><p>I think a section header like <code>Discussion</code> or <code>Practical considerations</code> fits better than <code>After thought</code>- to me, an afterthought means it's not important.</p><p>I like these summary paragraphs at the end to help the user understand what they have accomplished and how it may apply to a real use-case.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>